### ISSUE TITLE
test/Main.hs:7:1: error: Failed to load interface for ‘Katip.Tests.Format.Time’

### ISSUE THREAD

**Issue by juhp** [Soostone/katip#41]
```
Preprocessing test suite 'test' for katip-0.3.1.1...
:
test/Main.hs:7:1: error:
    Failed to load interface for ‘Katip.Tests.Format.Time’
    Perhaps you meant Katip.Format.Time (from katip-0.3.1.1)
    Use -v to see a list of the files searched for.
```

**Comment by MichaelXavier** [comment]
Fixed in katip-0.3.1.2. Could you try again?

**Comment by juhp** [comment]
Thanks
